packages/synapse-react-client/src/components/dataaccess/SubmissionPage/SubmissionPage.tsx

```diff
@@ -0,0 +1,542 @@
+  )
+}
+
+/**
+ * returns true iff the userId is a requester and/or listed on the accessor changes on the submission.
+ * @param userId
+ * @param submission
+ */
+function isRequesterOnSubmission(userId: string, submission: Submission) {
+  return (
+    submission.submittedBy === userId ||
+    Boolean(submission.accessorChanges.find(acc => acc.userId === userId))
+  )
+}
+
+/**
+ * Page for a Data Access Submission. Supports the following user types:
+ *  - ACT member or other designated reviewer can view, and choose to approve or reject.
+ *  - Data Access Request submitter can view and modify their own submission.
+ *  - Data Access Request requester (who is not a submitter) can view their own submission.
+ */
+export default function SubmissionPage(props: SubmissionPageProps) {
+  const { submissionId, isReviewer } = props
+  const [showRejectionDialog, setShowRejectionDialog] = useState(false)
+
+  const { data: currentUserProfile } = useGetCurrentUserProfile()
+
+  const { data: submission, isLoading: isLoadingSubmission } =
+    useGetDataAccessSubmission(submissionId)
+  const isRequester = Boolean(
+    submission &&
+      currentUserProfile &&
+      isRequesterOnSubmission(currentUserProfile.ownerId, submission),
+  )
+
+  const { data: accessApproval, isLoading: isLoadingAccessApproval } =
+    useGetUserAccessApproval(String(submissionId), {
+      enabled:
+        isRequester &&
+        submission &&
+        submission.state === SubmissionState.APPROVED,
+    })
+
+  const isExpired = Boolean(
+    accessApproval && dayjs(accessApproval.expiredOn).isBefore(dayjs()),
+  )
+
+  const isSubmitter =
+    submission && submission.submittedBy === currentUserProfile?.ownerId
+
+  const { data: accessRequirement } =
+    useGetAccessRequirements<ManagedACTAccessRequirement>(
+      parseInt(submission?.accessRequirementId!),
+      { enabled: !!submission },
+    )
+
+  const { data: acl, isLoading: isLoadingACL } = useGetAccessRequirementACL(
+    submission?.accessRequirementId!,
+    { enabled: !!submission, throwOnError: true },
+  )
+
+  const [showUpdateRequestDialog, setShowUpdateRequestDialog] = useState(false)
+  const [
+    showCancelRequestConfirmationDialog,
+    setShowCancelRequestConfirmationDialog,
+  ] = useState(false)
+  const [showApprovalConfirmation, setShowApprovalConfirmation] =
+    useState(false)
+
+  const navigate = useNavigate()
+
+  const reviewerIds = acl?.resourceAccess
+    .filter(ra => ra.accessType.includes(ACCESS_TYPE.REVIEW_SUBMISSIONS))
+    .map(ra => ra.principalId)
+
+  const displayedSubmissionState: ReactNode = useMemo(() => {
+    if (submission && !isLoadingAccessApproval) {
+      if (isExpired) {
+        return 'Expired'
+      }
+      return submission.state
+    }
+    return <Skeleton width={100} />
+  }, [isExpired, isLoadingAccessApproval, submission])
+
+  return (
+    <Grid2 container={true} spacing={4} className="SubmissionPage">
```

Comment:
Use MUI Grid2 to do some basic gridding--replace some bespoke CSS Grid with something more responsive.